app/products/calls/screens/call_screen/call_screen.tsx

```diff
@@ -1,5 +1,6 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
+/* eslint max-lines: off */
```

Comment:
maybe consider splitting this file into further components

Reply:
Yep, it's something we should consider for a clean-up prior to 1.0. That being said, I'm not totally convinced that arbitrary file length rules are always a good thing, if in order to get them you have to introduce more indirection, prop drilling, components, objects, garbage collection, etc. Sometimes a flat file is simpler, clearer, and more performant. As always, it's should be a case-by-case basis.